src/guides/v2.3/ui_comp_guide/components/ui-tab.md

```diff
@@ -35,3 +35,52 @@ See the [Admin Design Pattern Library (Tabs)]({{ page.baseurl }}/pattern-library
     <td><code>params.activeArea</code></td>
   </tr>
 </table>
+
+## Examples
+
+### Integrate the Tab component with the Form component
+
+This example defines the page layout
+
+```xml
```

Comment:
I believe we may remove this particular example (of page layout) since it's not directly related to integrating the tabs component into a form.

Thank you

Reply:
Removed page layout code

Thank you